src/__tests__/ModuleList/Article/Article.test.jsx

```diff
@@ -11,12 +25,58 @@ const RenderWithRouter = ({ children, moduleId, articleId }) => (
 );
 
 describe('The Article page', () => {
-  it('should render the same based on URL parameters', () => {
-    const tree = renderer.create(
-      <RenderWithRouter moduleId={42} articleId={64}>
+  beforeEach(() => {
+    getArticleText.mockClear();
+    getArticleMeta.mockClear();
+
+    getArticleMeta.mockResolvedValue(mockArticle.meta);
+    getArticleText.mockResolvedValue(mockArticle.text);
+  });
+  it('should match snapshot', async () => {
+    const { getByText, container } = render(
+      <RenderWithRouter articleId={articleId}>
+        <Article />
+      </RenderWithRouter>
+    );
+
+    await waitFor(() => expect(getByText(mockArticle.text)).toBeInTheDocument());
+    expect(container).toMatchSnapshot();
+  })
+  it('should get article text and meta for article id specified in the URL', async () => {
+    render(
+      <RenderWithRouter articleId={articleId}>
+        <Article />
+      </RenderWithRouter>
+    );
+
+    await waitFor(() =>  expect(getArticleMeta.mock.calls[0][0]).toBe(articleId));
```

Comment:
Very small thing: there are two spaces after the arrow(`=>`). Can you change it to one space?

Reply:
Sure thing! Done!